src/lib/viewers/archive/SearchBar.scss

```diff
@@ -0,0 +1,15 @@
+@import '~box-ui-elements/es/styles/variables';
+
+.bp-SearchBar {
+    display: flex;
+    flex: 0 0 70px;
+    align-items: center;
+    padding: 0 20px;
+    background: $almost-white;
+    border-bottom: 1px solid $bdl-gray-10;
+
+    input[type='search'] {
```

Comment:
Let's apply a class to the input so we can style it directly and avoid nesting our styles.

Reply:
That doesn't work cause we have to override `.be input[type='search']`

Reply:
Should we add a comment to that effect?